src/plugins/github/fetchReposForOrg.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import {type RepoId, makeRepoId} from "../../core/repoId";
+import dedent from "../../util/dedent";
+
+const GITHUB_GRAPHQL_SERVER = "https://api.github.com/graphql";
+async function ghquery(query, token): any {
+  const body = JSON.stringify({query});
+  const fetchOptions = {
+    method: "POST",
+    body,
+    headers: {
+      Authorization: `bearer ${token}`,
+    },
+  };
+  const response = await fetch(GITHUB_GRAPHQL_SERVER, fetchOptions);
+  const json = await response.json();
+  if (json.errors) {
+    throw json.errors;
+  }
+  return json;
+}
+
+export async function fetchReposForOrg(
+  org: string,
+  token: string
+): Promise<$ReadOnlyArray<RepoId>> {
+  const query = dedent`\
+  {
+    search(query: "org:${org}", type: REPOSITORY, first: 50) {
+      nodes {
+        ... on Repository {
+          name
+        }
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+      }
+    }
+  }
+  `;
+  let json = await ghquery(query, token);
+  console.log(json);
+  let repos = json.data.search.nodes.map((x) => makeRepoId(org, x.name));
+  while (json.data.search.pageInfo.hasNextPage) {
+    const query = dedent`\
```

Comment:
After you switch from string interpolation to the structured API, it
will also be easier to make this second query reuse the components of
the original. With the structured API, GraphQL queries and their
components are simple immutable values, rather than arbitrary sequences
of Unicode code points. Please take advantage of this to ensure that
you’re querying for the same selection set on the initial and subsequent
queries.